src/DocBlock/Tags/MethodParameter.php

```diff
@@ -60,8 +69,17 @@ public function isVariadic(): bool
         return $this->isVariadic;
     }
 
-    public function getDefaultValue(): ?string
+    public function getDefaultValue(): mixed
```

Comment:
Can we please revert this as I did notice before, because I see this as a backward compatibility break. I  think we should not do that right now. This will also solve the failing tests